applications/ConvectionDiffusionApplication/custom_strategies/strategies/explicit_runge_kutta_4_eulerian_convdiff_strategy.h

```diff
@@ -0,0 +1,385 @@
+
+    /// The local vector definition
+    typedef typename TDenseSpace::VectorType LocalSystemVectorType;
+
+    /** Counted pointer of ClassName */
+    KRATOS_CLASS_POINTER_DEFINITION(ExplicitSolvingStrategyRungeKutta4ConvectionDiffusion);
+
+    ///@}
+    ///@name Life Cycle
+    ///@{
+
+    /**
+     * @brief Default constructor. (with parameters)
+     * @param rModelPart The model part of the problem
+     * @param ThisParameters The configuration parameters
+     */
+    explicit ExplicitSolvingStrategyRungeKutta4ConvectionDiffusion(
+        ModelPart &rModelPart,
+        Parameters ThisParameters)
+        : BaseType(rModelPart, ThisParameters)
+    {
+    }
+
+    /**
+     * @brief Default constructor.
+     * @param rModelPart The model part to be computed
+     * @param pExplicitBuilder The pointer to the explicit builder and solver
+     * @param MoveMeshFlag The flag to set if the mesh is moved or not
+     */
+    explicit ExplicitSolvingStrategyRungeKutta4ConvectionDiffusion(
+        ModelPart &rModelPart,
+        typename ExplicitBuilderType::Pointer pExplicitBuilder,
+        bool MoveMeshFlag = false,
+        int RebuildLevel = 0)
+        : BaseType(rModelPart, pExplicitBuilder, MoveMeshFlag, RebuildLevel)
+    {
+    }
+
+    /**
+     * @brief Default constructor.
+     * @param rModelPart The model part to be computed
+     * @param MoveMeshFlag The flag to set if the mesh is moved or not
+     */
+    explicit ExplicitSolvingStrategyRungeKutta4ConvectionDiffusion(
+        ModelPart &rModelPart,
+        bool MoveMeshFlag = false,
+        int RebuildLevel = 0)
+        : BaseType(rModelPart, MoveMeshFlag, RebuildLevel)
+    {
+    }
+
+    /** Copy constructor.
+     */
+    ExplicitSolvingStrategyRungeKutta4ConvectionDiffusion(const ExplicitSolvingStrategyRungeKutta4ConvectionDiffusion &Other) = delete;
+
+    /** Destructor.
+     */
+    virtual ~ExplicitSolvingStrategyRungeKutta4ConvectionDiffusion() = default;
+
+    ///@}
+    ///@name Operators
+    ///@{
+
+
+    ///@}
+    ///@name Operations
+    ///@{
+
+    /**
+     * @brief Initialization of variables.
+     * @details In this method, we call the base strategy initialize and initialize the projection variable.
+     * This is required to prevent OpenMP errors as the projection variable is stored in the non-historical database.
+     */
+    void Initialize() override
+    {
+        KRATOS_TRY;
+
+        auto& r_model_part = BaseType::GetModelPart();
+        auto& r_process_info = r_model_part.GetProcessInfo();
+        // Call the base method
+        BaseType::Initialize();
+        // If required, initialize the OSS projection variables
+        if (r_process_info[OSS_SWITCH]) {
+            for (auto& r_node : r_model_part.GetCommunicator().LocalMesh().Nodes()) {
+                r_node.SetValue(SCALAR_PROJECTION, 0.0);
+            }
+        }
+        r_process_info.GetValue(TIME_INTEGRATION_THETA) = 1.0;
```

Comment:
Why do you need to initialize the `TIME_INTEGRATION_THETA` to one?

Reply:
I answered below.

Reply:
Now to zero.